org-code-javabuilder/lib/src/main/java/dev/javabuilder/LocalProjectFileManager.java

```diff
@@ -0,0 +1,31 @@
+package dev.javabuilder;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.code.javabuilder.ProjectFile;
+import org.code.javabuilder.ProjectFileManager;
+import org.code.javabuilder.UserFacingException;
+
+/** Loads the MyClass.java file from the resources folder. */
```

Comment:
is the plan with these `Local...` classes that they are temporary and will be removed once we load things from S3? Or are we going to keep them for future testing?

Reply:
I don't think they need to remain in this same format, but I'd like to keep them around in general. They made testing & development iteration much faster.

Reply:
Makes sense. It might be helpful to add a comment on each of these that they are for local testing only, I got a bit confused at first when I looked at them.

Reply:
Done!